weed/command/webdav.go

```diff
@@ -126,19 +129,23 @@ func (wo *WebDavOption) startWebDav() bool {
 
 	httpS := &http.Server{Handler: ws.Handler}
 
-	listenAddress := fmt.Sprintf(":%d", *wo.port)
+	if wo.bindIp == nil {
```

Comment:
when wo.bindIp can be nil?

Reply:
Here `WebDavOption` is also used in `command/server` and `command/filer`. If not assigned, it is nil.

The reason for not modifying these caller and performing a default fallback here is that even if `-ip/-ip.bind` is specified, both commands will still spawn webdav server listening on all interfaces. If following the semantic, it will change the behavior and may break existing setup.

If such breaking changes are fine, I can further modify the logic of server and filer and make the webdav servers of server and filer also bind to the port specified by `-ip/-ip.bind`.